src/test/java/org/smartregister/pathevaluator/plan/PlanEvaluatorTest.java

```diff
@@ -99,7 +99,7 @@ public void testEvaluatePlanForInactivePlan() {
 	}
 	
 	@Test
-	public void testEvaluatePlanEvaluatesConditions() {
```

Comment:
Retain this test and just  amend the trigger TriggerType.EVENT_SUBMISSION .

Add a new test that unit tests TriggerType.PLAN_ACTIVATION does queue the jobs